### PR TITLE
Code formatting and modification of code test database information

### PR DESCRIPTION
**Pull request by zhangchenglin**
Code formatting and modification of code test database information

### CODE REVIEW

**Comment by zhangchenglin** [comment]
https://github.com/PHPAuth/PHPAuth/pull/461/files?file-filters%5B%5D=.php#diff-ebd98368c673d521f95a7f40fff9ea0f20763fdd85dc6071fb7f710343804bcfL31-R33

File: tests/AuthTest.php [hunk 1]
```diff
@@ -28,7 +30,7 @@ public static function setUpBeforeClass()
         require_once __DIR__ . '/../Auth.php';
         require_once __DIR__ . '/../Config.php';
 
-        self::$dbh = new PDO("mysql:host=127.0.0.1;dbname=phpauthtest", "root", "");
+        self::$dbh = new PDO("mysql:host=127.0.0.1;dbname=phpauth_test_table", "phpauth_test_user", "");
```
Comment (zhangchenglin) [inline tests/AuthTest.php:33]: Clearer, no need to create a new test database separately

**Review by zhangchenglin — COMMENTED** [review]
Clearer, no need to create a new test database separately

**Comment by Conver** [comment]
@all-contributors please add zhangchenglin for Code

**Pull request merged**